types/blessed/index.d.ts

```diff
@@ -74,7 +74,7 @@ declare namespace Blessed {
                 /**
                  * Shape of the cursor. Can be: block, underline, or line.
                 */
-                shape: boolean;
+                shape: boolean|string;
```

Comment:
I can't find `boolean` anywhere in the source or docs. Also, why not make this `"block" | "underline" | "line"`?

Reply:
i changed them.